repos/fdbt-site/cypress_tests/cypress/support/passengerTypes.ts

```diff
@@ -101,7 +102,7 @@ export const createEditSinglePassengerTypes = (): void => {
 };
 
 export const createEditGroupPassengerTypes = (): void => {
-    addGroupPassengerType();
+    addGroupPassengerType('My Group');
```

Comment:
Should this not be lower case to maintain the existing test?

Reply:
Yes it should